### PR TITLE
Update submodule hafs_graphics.fd/hrd_gplot

### PR DESCRIPTION
**Pull request by ghassan-alaka**
## Description of changes
This PR brings the latest upgrades, fixes, and other modifications from HRD's GPLOT graphics package into the HAFS-Community `develop` branch. The functionality has been extensively tested in the HAFS workflow by HRD employees (@ghassan-alaka @AndrewHazelton @lgramer) and will further stabilize the hrdgraphics task.

## Dependencies (optional)
The HAFS-Community fork of GPLOT was successfully updated and can be found here:
-[ hafs-community/GPLOT/pull/11 ](https://github.com/hafs-community/GPLOT/pull/11)

## Contributors (optional)
Extensive testing and upgrades were performed by @AndrewHazelton, @lgramer, and Mu-Chieh Ko at HRD. A big thanks to @BinLiu-NOAA who has guided us through the merge process.

## Tests conducted
Extensive testing on Jet, Hera, and Orion. This package will NOT run on WCOSS2 due to issues with deprecated and/or unavailable software on that machine.

## Application-level regression test status
N/A

### CODE REVIEW

**Review by AndrewHazelton — APPROVED** [review]

**Comment by BinLiu-NOAA** [comment]
@ghassan-alaka, PR #171 is merged. Please sync this PR branch with the latest HAFS develop branch. Thanks! Meanwhile, there is no HAFS application/workflow level regression tests using hrd graphics yet. So, we will not need conduct regression tests for this PR.

**Comment by ghassan-alaka** [comment]
@BinLiu-NOAA I think this PR is almost ready to be merged. Within the `hafs-community/gplot` repository, all changes were merged into the `support/HAFS` branch. Two key updates to this PR today:
1) updated _.gitmodules_ to point to the correct GPLOT submodule branch
2) committed the updated GPLOT files under _./sorc/hafs_graphics.fd/hrd_gplot_

I am requesting that @AndrewHazelton or @lgramer conduct a test of this updated `support/HAFS` branch within the HAFS workflow. It should behave exactly as the `feature/realtime2022` branch, but it would be good to be sure that nothing unexpected occurred during the merge.

File: jobs/JHAFS_HRDGRAPHICS [hunk 1]
```diff
@@ -44,13 +44,14 @@ export CDATE=${CDATE:-${YMDH}}
...
 export STORMID=${STORMID:-00L}
+export pubbasin2=${pubbasin2:-AL}
 export ENSDA=${ENSDA:-NO}
 
 export INPdir=${INPdir:-${WORKhafs}/forecast}
 export DATA=${WORKhafs}/hrdgraphics
 
 export SENDCOM=${SENDCOM:-YES}
-export SCRUBDATA=${SCRUBDATA:-YES}
+export SCRUBDATA=${SCRUBDATA:-NO}
```
Comment (BinLiu-NOAA) [inline jobs/JHAFS_HRDGRAPHICS]: @ghassan-alaka, could you revert this change? The default value of SCRUBDATA should be YES, unless absolutely wanted to otherwise be NO.

**Review by ghassan-alaka — COMMENTED** [review]
I reverted the default definition of SCRUBDATA to "YES" for the hrdgraphics task. Now this task is consistent with behavior of other HAFS workflow tasks.

File: jobs/JHAFS_HRDGRAPHICS (hunk 1 above)
Comment (ghassan-alaka) [reply jobs/JHAFS_HRDGRAPHICS]: Agreed @BinLiu-NOAA . I reverted the change and commented on it below.

**Review by BinLiu-NOAA — APPROVED** [review]
Looks good to me.
